docs/source/daml-script/index.rst

```diff
@@ -212,7 +224,7 @@ We can then initialize our ledger passing in the json file via ``--input-file``.
 
 If you open Navigator, you can now see the contracts that have been created.
 
-While we will not use it here, there is also an ``--output-file``
+As mentioned, there is also an ``--output-file``
```

Comment:
Can we fold this into the paragraph above? It seems a bit weird to mention the option above without explaining it and then mention it somewhere down here.

Reply:
I've tried moving this up, which I think flows a little bit better. Thoughts?

Reply:
I like it, thanks!